treeherder/perf/models.py

```diff
@@ -81,9 +84,45 @@ def __str__(self):
                                     self.last_updated)
 
 
+class PerformanceDatumManager(models.Manager):
+    """
+    Convenience functions for operations on groups of performance datums
+    """
+
+    def cycle_data(self, repository, cycle_interval, chunk_size, sleep_time):
+        """Delete data older than cycle_interval, splitting the target data
+into chunks of chunk_size size."""
+
+        max_timestamp = datetime.datetime.now() - cycle_interval
```

Comment:
cycle_interval is set to `datetime.timedelta(days=cycle_interval)` in `treeherder/model/management/commands/cycle_data.py` :)